### PR TITLE
Rename smoke tests as per issue #3083

### PR DESCRIPTION
**Pull request by sthirugn**

### CODE REVIEW

**Comment by jyejare** [comment]
ACK

**Comment by sghai** [comment]
ACK

**Comment by sghai** [comment]
LGTM though leaving it open for others to review..

File: tests/foreman/smoke/test_smoke.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def test_selinux_foreman_module(self):
 
         self.assertEqual(rpm_version.replace('-', '.'), semodule_version)
 
-    def test_installer_check_services(self):
+    def test_positive_installer_check_services(self):
         """@Test: Services services start correctly
```
Comment (oshtaier) [inline tests/foreman/smoke/test_smoke.py]: `"""@Test: Services start correctly`

File: tests/foreman/smoke/test_smoke.py [hunk 2]
```diff
@@ -46,7 +46,7 @@ def test_selinux_foreman_module(self):
 
         self.assertEqual(rpm_version.replace('-', '.'), semodule_version)
 
-    def test_installer_check_services(self):
+    def test_positive_installer_check_services(self):
```
Comment (oshtaier) [inline tests/foreman/smoke/test_smoke.py]: Maybe `test_positive_check_installer_services`

File: tests/foreman/smoke/test_smoke.py [hunk 3]
```diff
@@ -108,7 +108,7 @@ def test_installer_check_services(self):
                 '{0} responded with {1}'.format(service, server_response)
             )
 
-    def test_installer_logfile_check(self):
+    def test_positive_installer_logfile_check(self):
```
Comment (oshtaier) [inline tests/foreman/smoke/test_smoke.py]: Maybe `test_positive_check_installer_logfile`

**Comment by oshtaier** [comment]
ACK pending comments

**Comment by sthirugn** [comment]
Comments updated and rebased.

**Pull request merged**